### PR TITLE
QE Feedback for CNV-1616 Editing VMs

### PR DESCRIPTION
**Pull request by bgaydosrh**
This content has been peer reviewed. Will squash and merge shortly.

For CNV 2.0

### CODE REVIEW

**Comment by bgaydosrh** [comment]
Please label branch/enterprise-4.1 and branch/enterprise-4.2

For CNV 2.0

File: modules/cnv-editing-vm-web.adoc [hunk 1]
```diff
@@ -20,7 +20,6 @@ Edit select values of a virtual machine in the web console or from the *Virtual
 . You can change the *Flavor* but only to *Custom*, which provides additional fields for *CPU* and *Memory*.
 . Click *Save*.
 
-The updated values are shown after the operation is processed.
-
-
+The values and virtual machine are updated after the operation is processed.
```
Comment (kalexand-rh) [inline modules/cnv-editing-vm-web.adoc]: @bgaydosrh, I think you need to nest both of these statements under the final step in the procedure. I also think that you should say "... changes do not display until you reboot the virtual machine." in the second line.

**Pull request merged**